File-Formats/DocIO/Convert-Word-Document-to-Image-in-Google-App-Engine.md

```diff
@@ -0,0 +1,261 @@
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 5: Include the following namespaces in the **HomeController.cs** file.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIORenderer;
+
+{% endhighlight %}
+{% endtabs %}
+
+Step 6: A default action method named Index will be present in HomeController.cs. Right click on Index method and select **Go To View** where you will be directed to its associated view page **Index.cshtml**.
+
+Step 7: Add a new button in the Index.cshtml as shown below.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+@{
+    Html.BeginForm("ConvertWordToImage", "Home", FormMethod.Get);
+    {
+        <div>
+            <input type="submit" value="Convert Word to image" style="width:185px;height:27px" />
+        </div>
+    }
+    Html.EndForm();
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+Step 8: Add a new action method **ConvertWordToImage** in HomeController.cs and include the below code snippet to **convert the Word document to image** and download it.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+using (FileStream inputStream = new FileStream(Path.GetFullPath("Data/Input.docx"), FileMode.Open, FileAccess.Read))
+{
+    //Creating a new document.
+    using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
+    {                     
+        //Creates a new instance of DocIORenderer class.
+        using (DocIORenderer render = new DocIORenderer())
+        {
+            //Converts the first page of word document to image
+            imageStream = (MemoryStream)document.RenderAsImages(0, ExportImageFormat.Jpeg);
+            imageStream.Position = 0;  
+            //Download image file in the browser.
+            return File(imageStream, "image/jpeg", "WordToimage_Page1.jpeg");                       
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Move application to App Engine
+
+Step 1: Open the **Cloud Shell editor**.
+![Cloud Shell Editor](GCP_Images/Cloud-Shell-Editor-WordtoImage.png)
+
+Step 2: Drag and drop the sample from your local machine to **Workspace**.
+![Open the Home Workspace](GCP_Images/Workspace-WordtoImage.png)
+
+N> If you have your sample application in your local machine, drag and drop it into the Workspace. If you created the sample using the Cloud Shell terminal command, it will be available in the Workspace.
+
+Step 3: Open the Cloud Shell Terminal and run the following **command** to view the files and directories within your **current Workspace**.
+
+{% tabs %}
+{% highlight c# tabtitle="CLI" %}
+
+$ ls
```

Comment:
$ symbol is used in certain places only. If needed add in all md files. If not needed remove from all md files.

Reply:
updated